module/vision/VisualMesh/src/VisualMesh.cpp

```diff
@@ -115,38 +115,44 @@ namespace vision {
 
 
             // -- Graphing for NUsight
-            // msg->classifications.emplace_back(results.classifications.front().first,
-            //                                   results.classifications.front().second);
-            // msg->classifications.emplace_back(results.classifications.back().first,
-            //                                   results.classifications.back().second);
+            msg->classifications.emplace_back(results.classifications.front().first,
+                                              results.classifications.front().second);
+            msg->classifications.emplace_back(results.classifications.back().first,
+                                              results.classifications.back().second);
 
 
-            // std::vector<std::tuple<Eigen::Vector2i, Eigen::Vector2i, Eigen::Vector4d>,
-            //             Eigen::aligned_allocator<std::tuple<Eigen::Vector2i, Eigen::Vector2i, Eigen::Vector4d>>>
-            //     lines;
+            std::vector<std::tuple<Eigen::Vector2i, Eigen::Vector2i, Eigen::Vector4d>,
+                        Eigen::aligned_allocator<std::tuple<Eigen::Vector2i, Eigen::Vector2i, Eigen::Vector4d>>>
+                lines;
 
 
-            // std::vector<std::array<int, 2>> pixel_coordinates = results.pixel_coordinates;
-            // std::vector<float> classification = results.classifications.front().second;
+            std::vector<std::array<int, 2>> pixel_coordinates;
+            std::vector<std::array<float, 2>> lazyConvert = results.pixel_coordinates;
+            pixel_coordinates.reserve(lazyConvert.size());
```

Comment:
This may not be a segfault but it's basically one.
You want resize if you're doing it like this, or push_back on line 133 if you're doing reserve